Source/QuartzCore additions/CGPathAdditions.m

```diff
@@ -21,7 +21,7 @@ CGFloat fixInfinity(CGFloat inputFloat){
     return inputFloat;
 }
 
-CGPoint *fixPointsInfinity(CGPathElement *element){
+CGPoint *fixPointsInfinity(const CGPathElement *element){
```

Comment:
I don't agree with it for keeping const validation and `void applier (void *info, const CGPathElement *element)` can not change to `..., CGPathElement *element)`.